### PR TITLE
Finish merging with the original Wyvern repository + Update the patches

### PR DESCRIPTION
**Pull request by WyvernPJ**
Up-to-date all the scripts with the original Wyvern repository.

### CODE REVIEW

**Comment by WyvernPJ** [comment]
if you merge it just copy directly the name of the commit, that is more readable than just `Merging pull request #n from something`

File: pom.xml [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+        <module>Wyvern-Server</module>
+        <module>Wyvern-API</module>
+    </modules>
 
     <build>
         <defaultGoal>clean install</defaultGoal>
     </build>
 
     <properties>
-        <maven.compiler.source>17</maven.compiler.source>
```
Comment (RackDevelopment) [inline pom.xml:24]: Please update compiler source to Java 17, rather than 16.

**Review by RackDevelopment — CHANGES_REQUESTED** [review]
Looks good besides the J16 mention rathe than 17 mentioned.

File: pom.xml [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-        <maven.compiler.target>17</maven.compiler.target>
+        <maven.compiler.source>16</maven.compiler.source>
+        <maven.compiler.target>16</maven.compiler.target>
```
Comment (RackDevelopment) [inline pom.xml]: Java 16 Call, should be J17

File: patches/api/0001-Wyvern-API-POM.patch [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+--- /dev/null
++++ b/pom.xml
+@@ -0,0 +1,54 @@
++<?xml version="1.0" encoding="UTF-8"?>
++<project xmlns="http://maven.apache.org/POM/4.0.0"
++         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
++         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
++    <modelVersion>4.0.0</modelVersion>
++
++    <groupId>com.wyvern</groupId>
```
Comment (RackDevelopment) [inline patches/api/0001-Wyvern-API-POM.patch]: Please revert back to tech.rackdevelopment.wyvern for the groupID, I am only pointing this one out but I am aware there are other files that contain it as well.

**Comment by WyvernPJ** [comment]
@RackDevelopment ill just gonna make a commit fixing these later if you are okay with me pushing it directly

**Comment by RackDevelopment** [comment]
Id prefer if you just add the commit here, if you cant, go ahead.

**Pull request merged**